popde/awkdepy.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+from density_estimate import SimpleKernelDensityEstimation
+from KDEpy.TreeKDE import TreeKDE
+
+class General_adaptiveKDE(SimpleKernelDensityEstimation):
```

Comment:
Suggest just 'AdaptiveBwKDE' - I don't know if this is more or less general than other possibilities .. 
Also, is there a reason why this does not subclass VariableBwKDE?  As far as I can see the fit_KDEpy and evaluate_KDEpy methods there could just be used here once self.bandwith has been set